lib/origin/selector.rb

```diff
@@ -18,7 +18,10 @@ class Selector < Smash
     def merge!(other)
       other.each_pair do |key, value|
         if value.is_a?(Hash) && self[key.to_s].is_a?(Hash)
-          value = self[key.to_s].merge(value)
+          value = self[key.to_s].merge(value) do |_key, oldval, newval|
+            return newval if !multi_variable?(_key)
+            (oldval + newval).uniq
```

Comment:
variables made up of more than one word should be snake-cased in Ruby. So you can call these variables old_val and new_val